content/en/docs/overview/_index.md

```diff
@@ -0,0 +1,19 @@
+---
+title: "Overview"
+weight: 200
+---
+
+In this chapter, we're going to get an overview which covers the most important language features of Go.
+This is only an overview and by no means a complete list.
```

Comment:
IMHO this paragraph is a bit long and windy. We already said `the most important language features`. I think we can shorten this. The course is called `go-basics`.

Reply:
Removed some sentences from this paragraph.